### PR TITLE
Added Roadrash cheat codes

### PR DESCRIPTION
**Pull request by gautamkrishnar**
**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.cheatcc.com/pc/roadrash.html

**Why did you choose this data source?**
relevant 

**Are there any other alternative (better) data sources?**
none

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
gamers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Are you having any problems? Do you need our help with anything?**
none

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
ddg

---

IA Page: https://duck.co/ia/view/road_rash_cheat_sheet

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@moollaza check out this!

**Comment by jagtalon** [comment]
@gautamkrishnar checking it out!

**Comment by jagtalon** [comment]
@gautamkrishnar I'm a bit hesitant about this because it doesn't trigger on the query that it's intended for namely "roadrash cheat codes" and only triggers on "roadrash cheat sheets" and the like. Will talk to the team about this and get back to you.

**Comment by gautamkrishnar** [comment]
Okay that may be some problem with the filename or id. I ll be happy to make any changes if needed. Thanks :+1: @jagtalon

File: share/goodie/cheat_sheets/json/roadrash.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Road Rash",
+    "id": "roadrash_cheat_sheet",
+    "metadata": {
+        "description": "First while playing type xyzzy then type the following cheats",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/roadrash.json]: Please move this out of `metadata` it belongs at the top level

File: share/goodie/cheat_sheets/json/roadrash.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "sourceName": "Cheatcc",
+        "sourceUrl": "http://www.cheatcc.com/pc/roadrash.html"
+    },
+    "template_type": "keyboard",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/roadrash.json]: Let's use the `reference` template here

File: share/goodie/cheat_sheets/json/roadrash.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Road Rash",
+    "id": "roadrash_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/roadrash.json]: Let's change this to `road_rash_cheat_sheet`. Likewise, let's rename the file `road-rash.json`. It looks like the name is typically spelled with a space between the words.

We can add `"aliases" : [ "roadrash" ],` if you'd like `roadrash cheat sheet` to trigger this as well

**Comment by gautamkrishnar** [comment]
@moollaza i had removed the contents out of metadata block but it is now not deploying successfully to the beta.duckduckgo.com

File: share/goodie/cheat_sheets/json/roadrash.json (hunk 1 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/roadrash.json]: Is it any recent changes made to the code? @moollaza 
I had actually placed the description and other keys inside the metadata for all my old PRs

**Comment by moollaza** [comment]
@gautamkrishnar you only need to move `description` out of `metadata`; `sourceName` and `sourceUrl` still belong inside there

**Comment by gautamkrishnar** [comment]
@moollaza done :+1:

**Comment by gautamkrishnar** [comment]
@moollaza :+1: Fixed

File: share/goodie/cheat_sheets/json/road-rash.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Road Rash",
+    "id": "road_rash_cheat_sheet",
+    "description": "First while playing type xyzzy then type the following cheats",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/road-rash.json]: Let's improve the phrasing here:

`While playing, enter "XYZZY", then enter one of the following codes"

**Comment by moollaza** [comment]
@gautamkrishnar Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@gautamkrishnar congrats! It's live: https://duckduckgo.com/?q=road+rash+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**